### PR TITLE
Update merge_sort_fastest.py

### PR DESCRIPTION
**Pull request by yesIamHasi**
Thank you very much for pointing that out. I am really sorry I accidentally uploaded the wrong file last time. The changes you made didn't worked for some inputs. This one works fine.

### CODE REVIEW

**Comment by harshildarji** [comment]
Hello @yesIamHasi, It would be really helpful if you could you please let use know about the inputs for which current code is not working as expected?

File: sorts/merge_sort_fastest.py [hunk 1]
```diff
@@ -4,16 +4,51 @@
...
+    plt.show()
+    plt.plot(SORTED_LIST)
+    plt.xlabel('Merge Sorted Data')
+    plt.show()
+
+if __name__ == '__main__':
+    # Basic Test
+    LIST = [0,5,2,4,3,9,1,7,6,8]
+    print ('Unordered list {}'.format(LIST))
+    print ('Merge sorted list {}'.format(merge_sort(LIST)))
```
Comment (gabru-md) [inline sorts/merge_sort_fastest.py]: can we store the `merge_sort(LIST)` into some variable and then use it at multiple places?

File: sorts/merge_sort_fastest.py [hunk 2]
```diff
@@ -4,16 +4,51 @@
 Best Case Scenario : O(n)
 Worst Case Scenario : O(n)
 '''
+# These modules are required for testing only.
+import random, copy
+import matplotlib.pyplot as plt
```
Comment (AnshulMalik) [inline sorts/merge_sort_fastest.py]: Umm, it's nice to see the output as graph, it really helps in visualising the data. But I think it's not required here. We can just test for the output.

File: sorts/merge_sort_fastest.py [hunk 3]
```diff
@@ -4,16 +4,51 @@
...
+            continue
     end.reverse()
-    return (start + end)
+    return start + end
+
+def visualize(LIST, SORTED_LIST):
+    '''
+    Plots unsorted and sorted data.
+    '''
+    plt.plot(LIST)
```
Comment (christianbender) [inline sorts/merge_sort_fastest.py]: Changed the code like this:
```python
figure1 = plt.figure(1)
plt.plot(LIST)
plt.xlabel('Random Data')
figure1.show()
figure2 = plt.figure(2)
plt.plot(SORTED_LIST)
plt.xlabel('Merge Sorted Data')
figure2.show()

raw_input() # keeps the figures alive
```

With the code above it show up two figures parallel at the same time.

File: sorts/merge_sort_fastest.py (hunk 2 above)
Comment (christianbender) [reply sorts/merge_sort_fastest.py]: @AnshulMalik 
good point! The function ```min``` and ```max``` have O(n). And this runtime is dominant.

**Pull request closed without merging**